api/lb3app/server/config.production.json

```diff
@@ -1,22 +0,0 @@
-{
-  "restApiRoot": "/api",
-  "host": "0.0.0.0",
-  "port": 4000,
-  "remoting": {
-    "context": false,
-    "rest": {
```

Comment:
The main reason deployment failed yesterday, was this file was lingering around. 

I had updated `config.json` but this was more specific to the env.